### PR TITLE
Commands: create_foodStorage

### PR DESCRIPTION
**Pull request by leezahuor**
# Issue/Task:
https://fridgeventory.atlassian.net/browse/FRID-28
Updated the current `create_freezer()` method in commands to be more generic `create_foodStorage(foodStorage_type)`. 

# Checklist
- [X] Fixed Issue/Accomplished Task
- [X] Wrote tests
- [X] All test cases passing

### CODE REVIEW

File: tests/Session/test_commands.py [hunk 1]
```diff
@@ -41,39 +42,39 @@ def testCreateFreezer(self, mocked_input):
...
-        capturedPrintOutput = io.StringIO()
-        sys.stdout = capturedPrintOutput
+        ## new food storage with existing name, invalid response, abort ##
+        # Grab print output from calling list()
+        capturedListOutput = io.StringIO()
+        sys.stdout = capturedListOutput
 
-        mocked_input.side_effect = [freezer_name, "x", "n"]
-        self.commands.create_freezer()
+        mocked_input.side_effect = [foodstorage_name, "x", "n"]
```
Comment (amandate) [inline tests/Session/test_commands.py:58]: I feel like "x" and "n" should be swapped for some constants

File: tests/Session/test_commands.py [hunk 2]
```diff
@@ -41,39 +42,39 @@ def testCreateFreezer(self, mocked_input):
...
-        capturedPrintOutput = io.StringIO()
-        sys.stdout = capturedPrintOutput
+        ## new food storage with existing name, yes ##
+        # Grab print output from calling list()
+        capturedListOutput = io.StringIO()
+        sys.stdout = capturedListOutput
 
-        mocked_input.side_effect = [freezer_name, "y"]
-        self.commands.create_freezer()
+        mocked_input.side_effect = [foodstorage_name, "y"]
```
Comment (amandate) [inline tests/Session/test_commands.py:74]: "y" swapped for a constant

**Review by amandate — COMMENTED** [review]
Just 1 small change!

File: tests/Session/test_commands.py (hunk 2 above)
Comment (leezahuor) [reply tests/Session/test_commands.py]: Done!

File: tests/Session/test_commands.py (hunk 1 above)
Comment (leezahuor) [reply tests/Session/test_commands.py]: Done!

**Review by amandate — APPROVED** [review]

**Pull request merged**